test/built-ins/Atomics/pause/non-integral-iterationnumber-throws.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2024 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-atomics.pause
+description: Atomics.pause throws on non-integral Number argument values
+features: [Atomics.pause]
+---*/
+
+const values = [
```

Comment:
this file doesn't have any non-integral numbers in it

Reply:
Consider also `true`, `"42"` and `{ valueOf: () => 42 }` to make sure that implementations don't treat this as ToNumber.

Reply:
Done.